dev/src/lobster/3dgrid.h

```diff
@@ -69,6 +70,24 @@ template<typename T> class Chunk3DGrid : NonCopyable {
         }
         dim = ndim;
     }
+
+    void Rotate(const matrix<int, 3, 3> &m) {
```

Comment:
I guess this is different in setup from what goes on in `cg_rotate` ? Can you check? Fine to keep this copy if its messy to share.

Reply:
Yeah, it's different in that `cg_rotate` only rotates around Z by 90 degrees. We could probably make that one use this logic, though